lib/pathcache.js

```diff
@@ -113,6 +113,12 @@ function resolvers(options, webpackResolver) {
       // Follow the webpack stylesheet idiom of '~path' meaning a path in a
       // modules folder and a unprefixed 'path' meaning a relative path like
       // './path'.
+      // except if you using requireSyntax, in this case you can omit ~
+      if (options.requireSyntax) {
```

Comment:
I'm not sure we should support this. It seems like a major break from normal webpack css preprocessing paths for disambiguating a ambiguaty in css paths. Maybe we can support it with a more clear name like `preferPackagePaths` or `autoPrefixPackagePaths`.

@shama thoughts?

@timaschew can you please add a test?